RA/mixfile.h

```diff
@@ -45,9 +45,9 @@ class MixFileClass : public Node<MixFileClass<T> >
 		static void const * Retrieve(char const *filename);
 
 		struct SubBlock {
-			int32_t CRC;				// CRC code for embedded file.
-			int32_t Offset;			// Offset from start of data section.
-			int32_t Size;				// Size of data subfile.
+			uint32_t CRC;				// CRC code for embedded file.
+			uint32_t Offset;			// Offset from start of data section.
+			uint32_t Size;				// Size of data subfile.
```

Comment:
Don't think CRC needs to change, and I've seen enough weird bugs there to be concerned...

(Not that I want to think about >2GB mixfiles, but the other fields being unsigned seems reasonable.)